### PR TITLE
More test actions

### LINKED ISSUE #68: Have test DAO start with a proposal in each stage
One queued, one pending, one boosted and one executed

### LINKED ISSUE #69: Give all default account some GEN budget on ganache
These accounts already get REP, let's give them sone GEN too for testing

### LINKED ISSUE #72: Test DAOs should have different names
this is too confusing:
image

### PR DESCRIPTION
**Pull request by ben-kaufman**
Fix #68 
Fix #69 
Fix #72

### CODE REVIEW

File: utils.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+var nouns = ['ninja', 'chair', 'pancake', 'statue', 'unicorn', 'rainbows', 'laser', 'senor', 'bunny', 'captain', 'nibblets', 'cupcake', 'carrot', 'gnomes', 'glitter', 'potato', 'salad', 'toejam', 'curtains', 'beets', 'toilet', 'exorcism', 'stick figures', 'mermaid eggs', 'sea barnacles', 'dragons', 'jellybeans', 'snakes', 'dolls', 'bushes', 'cookies', 'apples', 'ukulele', 'kazoo', 'banjo', 'opera singer', 'circus', 'trampoline', 'carousel', 'carnival', 'locomotive', 'hot air balloon', 'animator', 'artisan', 'artist', 'colorist', 'inker', 'coppersmith', 'director', 'designer', 'flatter', 'stylist', 'leadman', 'limner', 'make-up artist', 'model', 'musician', 'penciller', 'producer', 'scenographer', 'silversmith', 'teacher', 'beader', 'foreman', 'maintenance', 'engineering', 'mechanic', 'miller', 'moldmaker', 'panel beater', 'patternmaker', 'plant operator', 'plumber', 'sawfiler', 'shop foreman', 'soaper', 'wheelwright', 'woodworkers']
+
+const capitalize = (s) => {
+  if (typeof s !== 'string') return ''
+  return s.charAt(0).toUpperCase() + s.slice(1)
+}
+
+exports.generateRnadomName = function () {
```
Comment (orenyodfat) [inline utils.js]: why need this randomness ?

**Comment by orenyodfat** [comment]
Does this PR will break subgraph ?

File: utils.js (hunk 1 above)
Comment (ben-kaufman) [reply utils.js]: #72

**Comment by ben-kaufman** [comment]
Well it slightly changes the migration.json structure so theoretically yes.

**Comment by orenyodfat** [comment]
So the name of the test days are now random..
So to validate the correctness of the dao name one should compare that against the migration.json ...and not against hardcoded...right ?

**Comment by ben-kaufman** [comment]
Yes

**Pull request merged**